### PR TITLE
feat: add latest activity rail

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-1222] 

This PR is blocked by: https://github.com/artsy/metaphysics/pull/5936

### Description
This PR adds the latest activity rail home view section. 

Still todo for tomorrow: 
- Add tests
- Add href

https://github.com/user-attachments/assets/02d6d02a-75b2-4acc-8c24-664b691565f8

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- add latest activity rail - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1222]: https://artsyproduct.atlassian.net/browse/ONYX-1222?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Home/Home.tsx [hunk 1]
```diff
@@ -505,7 +505,7 @@ export const HomeFragmentContainer = memo(
       `,
       notificationsConnection: graphql`
         fragment Home_notificationsConnection on Viewer {
-          ...ActivityRail_notificationsConnection @arguments(count: 10)
+          ...ActivityRail_viewer @arguments(count: 10)
```
Comment (anandaroop) [inline src/app/Scenes/Home/Home.tsx:508]: **question (non-blocking):** So this (and the associated relay noise) was to correct a pre-existing naming glitch, yes?

**Review by anandaroop — APPROVED** [review]
👍🏽

File: src/app/Scenes/Home/Home.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/app/Scenes/Home/Home.tsx:508]: yes, it should have been `_viewer` instead

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (add latest activity rail - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 68f598382abca2d293b1cad361eaead6752f6bb8
</p>

**Pull request merged**